sql/src/test/java/io/crate/planner/operators/PushDownTest.java

```diff
@@ -26,21 +26,26 @@
 import io.crate.planner.TableStats;
 import io.crate.test.integration.CrateDummyClusterServiceUnitTest;
 import io.crate.testing.SQLExecutor;
+import org.junit.Before;
 import org.junit.Test;
 
 import static io.crate.planner.operators.LogicalPlannerTest.isPlan;
 
 public class PushDownTest extends CrateDummyClusterServiceUnitTest {
 
-    @Test
-    public void testOrderByOnUnionIsMovedBeneathUnion() {
-
-        TableStats tableStats = new TableStats();
+    private static TableStats tableStats;
+    private static SQLExecutor sqlExecutor;
```

Comment:
our test infrastructure contains memory-leak checks and these here will leak, counting towards the limit. Please make those non-static to avoid that

Reply:
Yeah, but phrasing... it's like code optimization, sorry